components/ResourceCreation/ResourceCreationPanel.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Center, Grid, Loader } from '@mantine/core';
+import React, { Suspense, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import { patientTestCaseState } from '../../state/atoms/patientTestCase';
+import { selectedPatientState } from '../../state/atoms/selectedPatient';
+import PatientCreation from './PatientCreation';
+import TestResourcesDisplay from './TestResourcesDisplay';
+
+export default function ResourceCreationPanel() {
+  const [isPatientModalOpen, setIsPatientModalOpen] = useState(false);
+  const [currentPatient, setCurrentPatient] = useState<string | null>(null);
+  const selectedPatient = useRecoilValue(selectedPatientState);
+  const currentPatients = useRecoilValue(patientTestCaseState);
+
+  const openModal = (patientId?: string) => {
+    if (patientId && Object.keys(currentPatients).includes(patientId)) {
+      setCurrentPatient(patientId);
+    } else {
+      setCurrentPatient(null);
+    }
+
+    setIsPatientModalOpen(true);
+  };
+  const closeModal = () => {
+    setIsPatientModalOpen(false);
+    setCurrentPatient(null);
+  };
+  return (
+    <>
+      <Center>
+        <div style={{ paddingTop: '24px', paddingBottom: '24px' }}>
+          <Button onClick={() => openModal()}>Create Test Patient</Button>
+        </div>
+      </Center>
+      <Grid>
+        {selectedPatient !== null && (
```

Comment:
I guess the way we did this, it causes the patient records to be pushed over and resized even if the measure bundle is not uploaded. Maybe we can add a boolean check to both this and the below Grid.Col to make sure that the columns only resize when measureBundle.content is not null?